spec/player_spec.rb

```diff
@@ -1,34 +1,33 @@
 require './lib/player'
 
 describe Player do
-  describe "#initialize" do
-    let(:player1){ Player.new("Ramin", 'x')}
+  describe '#initialize' do
+    let(:player1) { Player.new('Ramin', 'x')}
     it "return the player's name" do
-      expect(player1.name).to eql("Ramin")
+      expect(player1.name).to eql('Ramin')
     end
 
     it "return the player's tag" do
-      expect(player1.tag).to eql("x")
+      expect(player1.tag).to eql('x')
     end
 
-    it "return if the player is winner or not" do
+    it 'return if the player is winner or not' do
       expect(player1.is_winner).to eql(false)
     end
```

Comment:
## Positive scenario

- Specs cover positive and negative scenarios ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))
Example: there is a spec for correct usage of the method [e.g. it "creates an object if all required attributes are passed"]
and also there is spec for the wrong usage of a method  [e.g. it "does not create the object if required attributes are missing"].

  - [ ] Thus, please add positive scenario

Example:

```ruby
#positive scenario
  it 'returns true if the player is not the winner'' do
#negative scenario
  it 'returns false if the player is not the winner' do
```